src/load/azext_load/data_plane/utils/utils.py

```diff
@@ -347,6 +358,24 @@ def convert_yaml_to_test(data):
                 new_body["passFailCriteria"]["passFailMetrics"][metric_id][
                     "requestName"
                 ] = name
+    if data.get("autoStop") is not None:
+        if (isinstance(data["autoStop"], str)):
+            new_body["autoStopCriteria"] = {
+                "autoStopDisabled": True,
+            }
```

Comment:
I hope it will just patch and will not reset the errorPercentage and timeWindow ?

Reply:
the service does not patch, it resets the autoStopCriteria object
request object =>
![image](https://github.com/user-attachments/assets/f2aec162-3d31-4aad-a440-48e03616babc)
response object =>
![image](https://github.com/user-attachments/assets/6e0ca19b-5b23-42a1-a715-9ee5ac069ba4)

Reply:
I had been backfilling the complete autoStopCriteria object, but have updated now to backfill individual fields from the existing get test body